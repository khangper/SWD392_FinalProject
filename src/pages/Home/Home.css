.home-container {
  display: flex;
  justify-content: space-between;
  flex-direction: column;
}
.home-top{
  display: flex;
  justify-content: space-between;
  width: 100%;
}
.home-left {
  flex: 0 1 76%;
  padding: 0 20px 0 20px;
  background-color: #f7f7f7;
  overflow: hidden;
}

.home-right {
  flex: 0 1 24%;
  background-color: #f7f7f7;
}
.home-bottom{
  width: 100%;
  padding: 20px;
  background-color: #f7f7f7;
}

.live-streams-section,
.featured-courses-section,
.popular-instructors-section {
  margin-bottom: 20px;
  width: 100%;
}

.live-streams-container,
.featured-courses-container,
.popular-instructors-container {
  position: relative;
  display: flex;
  align-items: center;
  overflow: hidden;
  width: 100%;
  height: 100%;
}
.live-streams-header,
.featured-courses-header,
.popular-instructors-header {
  display: flex;
  justify-content: space-between;
  align-items: center;
}
.live-streams-title,
.featured-courses-title,
.popular-instructors-title {
  margin-top: 20px;
  font-size: 18px;
  margin-bottom: 20px;
}
.live-streams,
.featured-courses,
.popular-instructors {
  display: flex;
  overflow-x: auto;
  scroll-behavior: smooth;
  gap: 20px;
}
.live-streams::-webkit-scrollbar,
.featured-courses::-webkit-scrollbar,
.popular-instructors::-webkit-scrollbar,
.student-thought::-webkit-scrollbar {
  display: none;
}
.stream-card {
  border-radius: 3px;
  padding: 15px;
  background-position: center;
  background: rgba(51, 1, 51, 0.1);
  text-align: center;
  width: 160px;
  height: 170px;
  flex-shrink: 0;
}

.stream-link {
  text-align: center;
  display: inline-block;
  width: 100%;
  padding: 15px 10px;
  border-radius: 3px;
  background: rgba(255, 255, 255, 0.06);
  backdrop-filter: blur(30px);
  box-shadow: 0 6px 25px rgba(0, 0, 0, 0.13);
}
.stream-card img {
  width: 55px;
  height: 55px;
  border-radius: 50%;
}
.stream-card h4 {
  margin-top: 10px;
  margin-bottom: 5px;
  font-size: 14px;
  font-weight: 500;
  font-family: "Roboto", sans-serif;
  text-align: center;
  color: #333;
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
}
.stream-card p {
  font-family: "Roboto", sans-serif;
  font-weight: 400;
  color: #686f7a;
  display: inline-block;
  text-align: center;
  position: relative;
}
.stream-card span {
  width: 6px;
  height: 6px;
  background: #ed2a26;
  position: absolute;
  border-radius: 100%;
  top: 2px;
  right: -9px;
}

.scroll-button {
  background-color: transparent rgba(0, 0, 0, 0.13);
  border: none;
  border-radius: 3px;
  width: 30px;
  height: 30px;
  display: flex;
  align-items: center;
  justify-content: center;
  cursor: pointer;
  position: absolute;
  z-index: 1;
  top: 50%;
  transform: translateY(-50%);
  transition: background 0.3s ease;
}
.scroll-button:hover {
  background: #ff0000;
}

.scroll-button::before {
  content: "\2039";
  font-size: 1.5rem;
  color: #333;
  transition: color 0.3s ease;
}

.scroll-button.right::before {
  content: "\203A";
}

.scroll-button:hover::before {
  color: #fff;
}

.scroll-button.left {
  left: 0;
}

.scroll-button.right {
  right: 0;
}
.course-card,
.popular-instructors-card {
  position: relative;
  border-radius: 3px;
  background: #fff;
  width: 100%;
  padding: 10px;
  border: 1px solid #efefef;
  transition: all 0.2s ease-in-out;
}
.course-card:hover,
.popular-instructors-card:hover {
  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
}
.course-overlay {
  position: absolute;
  top: 10px;
  padding: 10px;
  left: 10px;
  right: 15px;
  bottom: 180px;
}
.badge-seller {
  border-radius: 3px;
  font-size: 10px;
  position: absolute;
  padding: 2px 10px;
  background: #fa8305;
  color: #fff;
  font-family: "Roboto", sans-serif;
  font-weight: 600;
  right: 0;
  top: 10px;
  text-transform: uppercase;
  box-shadow: 0 0 1px 1px rgba(20, 23, 28, 0.1),
    0 3px 1px 0 rgba(20, 23, 28, 0.1);
}
.course-review {
  color: #fff;
  background: #fdcc0d;
  display: inline-block;
  padding: 3px 15px 3px 10px;
  border-radius: 3px;
  font-size: 14px;
  font-weight: 500;
  font-family: "Roboto", sans-serif;
}
.course-timer {
  bottom: 0;
  position: absolute;
  margin-bottom: 10px;
  right: 0;
  margin-right: 10px;
  color: #fff;
  background: rgba(51, 51, 51, 0.8);
  display: inline-block;
  padding: 3px 10px;
  border-radius: 3px;
  font-size: 12px;
  font-weight: 500;
  font-family: "Roboto", sans-serif;
}
.course-card img {
  width: 260px;
  height: 150px;
  border-radius: 5px;
}
.course-card h4,
.popular-instructors-card h4 {
  margin-top: 10px;
  margin-bottom: 5px;
  font-size: 14px;
  font-weight: 500;
  font-family: "Roboto", sans-serif;
  color: #333;
  text-overflow: ellipsis;
  white-space: break-spaces;
  overflow: hidden;
}
.course-title {
  display: block;
  font-size: 16px;
  font-weight: 600;
  font-family: "Roboto", sans-serif;
  color: #333;
  text-align: left;
  margin-bottom: 7px;
  line-height: 24px;
}

.course-view-and-date {
  margin-bottom: 10px;
  font-size: 12px;
  margin-top: 5px;
  font-weight: 400;
  font-family: "Roboto", sans-serif;
  color: #686f7a;
  text-align: left;
  display: block;
}

.course-category {
  display: block;
  font-size: 13px;
  font-weight: 400;
  font-family: "Roboto", sans-serif;
  color: #686f7a;
  text-align: left;
  padding-top: 2px;
}

.course-info {
  float: left;
  width: 100%;
  margin-top: 15px;
}

.course-author {
  margin-bottom: 0;
  /* margin-top: 18px; */
  font-size: 14px;
  font-family: "Roboto", sans-serif;
  float: left;
  color: #686f7a;
  text-align: left;
  line-height: 24px;
}

.course-author a {
  color: #333;
  font-weight: 500;
}

.course-price {
  float: right;
  font-size: 18px;
  font-weight: 500;
  font-family: "Roboto", sans-serif;
  color: #333;
  text-align: right;
  /* margin-top: 18px; */
}

.course-more-dropdown {
  position: relative;
  display: inline-block;
}

.course-details {
  padding: 10px 5px;
  float: left;
  width: 100%;
  position: relative;
}

.course-details-header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  margin-bottom: 5px;
}

.course-more-dropdown a {
  font-size: 20px;
  color: #afafaf !important;
  text-decoration: none;
}

.course-more-dropdown:hover .course-more-dropdown-menu {
  display: block;
}

.course-more-dropdown-menu {
  display: none;
  position: absolute;
  background-color: #fff !important;
  min-width: 160px;
  box-shadow: 0 2px 3px 0 rgba(34, 36, 38, 0.15);
  border: 1px solid rgba(34, 36, 38, 0.15);
  z-index: 1;
  right: 0;
  border-radius: 3px;
}

.course-more-dropdown-menu span {
  color: #686f7a;
  font-size: 14px;
  font-weight: 400;
  font-family: "Roboto", sans-serif;
  padding: 10px;
  text-decoration: none;
  display: block;
  cursor: pointer;
}

.course-more-dropdown-menu span:hover {
  background-color: #f1f1f1;
}

.home-info-section {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  gap: 20px;
  margin-top: 40px;
}

.home-info-card {
  background: #fff;
  border: 1px solid #ddd;
  border-radius: 5px;
  padding: 20px;
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  width: calc(50% - 10px);
  text-align: center;
}

.home-info-icon-container {
  background: #ffecec;
  padding: 15px 5px 15px 5px;
  border-radius: 5px;
  display: inline-block;
  margin-bottom: 10px;
}

.home-info-card img {
  width: 40px;
  height: 40px;
}

.home-info-card h3 {
  margin: 10px 0;
  font-size: 18px;
  color: #333;
}

.home-info-card p {
  font-size: 14px;
  color: #686f7a;
  margin: 0;
}
.popular-instructor-image,
.instructor-profile-image {
  width: 100%;
  float: left;
  padding: 20px;
  text-align: center;
}
.popular-instructors-card img {
  width: 80px;
  height: 80px;
  border-radius: 50%;
  margin-bottom: 10px;
}

.popular-instructor-photo {
  text-align: center;
  width: 100px !important;
  height: 100px !important;
  display: inline-block !important;
  border-radius: 100%;
  border: 2px solid #fff;
  box-shadow: 0px 2px 2px 0px rgba(0, 0, 0, 0.1);
}
.popular-instructor-content,
.instructor-profile-content {
  text-align: center;
  float: left;
  width: 100%;
  margin-bottom: 15px;
  padding: 0 10px;
}
.popular-instructor-profile {
  display: flex;
  align-items: center;
  justify-content: center;
  text-align: center;
}
.popular-instructor-name,
.instructor-name {
  font-size: 16px;
  font-weight: 500;
  font-family: "Roboto", sans-serif;
  color: #333 !important;
  text-align: center;
  position: relative;
}
.verified-badge {
  display: inline-block;
  width: 15px;
  height: 15px;
  background-color: rgb(252, 252, 252); /* Twitter blue color */
  border-radius: 50%;
  position: relative;
  border: 2px solid #1da1f2;
  margin-left: 5px;
}

.verified-badge::before {
  top: 5px;
  left: 6px;
  width: 2px;
  height: 5px;
  border: solid #1da1f2;
  border-width: 0 2px 2px 0;
  transform: translate(-50%, -50%) rotate(45deg);
  content: "";
  position: absolute;
}
.popular-instructor-title {
  font-size: 13px;
  color: #686f7a;
  margin-top: 10px;
  font-family: "Roboto", sans-serif;
  font-weight: 400;
  margin-bottom: 17px;
  width: 250px;
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
}
.social-icons {
  list-style: none;
  padding: 0;
  margin: 0;
  display: flex;
  justify-content: center;
  gap: 5px;
}
.social-icons a {
  display: inline-block !important;
  margin-right: 5px;
  vertical-align: middle;
  margin-bottom: 0 !important;
}
.social-icons .sc-fb,
.social-icons .sc-tw,
.social-icons .sc-ln,
.social-icons .sc-yt {
  font-size: 20px;
  float: none;
  width: 35px;
  height: 36px;
  border-radius: 5px;
  text-align: center;
  line-height: 40px;
  padding: 9px 0;
  color: #fff;
  transition: all 0.3s ease-in-out;
}
.sc-fb {
  background: #3b5998;
}
.sc-tw {
  background: #1da1f2;
}
.sc-ln {
  background: #8d6cab;
}
.sc-yt {
  background: #ff0000;
}
.popular-instructor-stats {
  font-size: 14px;
  margin-top: 20px;
  font-weight: 400;
  font-family: "Roboto", sans-serif;
  color: #686f7a;
  text-align: center;
  display: block;
}

.home-instructor-profile{
  margin-top: 15px;
  margin-bottom: 25px;
  background: #fff;
  width: 100%;
  float: left;
  padding: 10px;
  border-radius: 3px;
  border: 1px solid #efefef;
  transition: all 0.2s ease-in-out;
}
.home-livestream-setup,
.home-featured-categories {
  margin-bottom: 25px;
  background: #fff;
  width: 100%;
  float: left;
  padding: 10px;
  border-radius: 3px;
  border: 1px solid #efefef;
  transition: all 0.2s ease-in-out;
}

.instructor-profile-header {
  display: flex;
  align-items: center;
  justify-content: center;
  text-align: center;
}
.home-instructor-profile-title {
  font-size: 13px;
  color: #686f7a;
  margin-top: 10px;
  font-family: "Roboto", sans-serif;
  font-weight: 400;
  margin-bottom: 17px;
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
}
.instructor-profile-stats {
  font-size: 12px;
  margin-top: 20px;
  font-weight: 400;
  font-family: "Roboto", sans-serif;
  color: #686f7a;
  text-align: center;
  display: block;
}
.instructor-profile-redirect {
  float: left;
  width: 100%;
  margin-top: 15px;
  font-size: 14px;
  font-weight: 500;
  font-family: "Roboto", sans-serif;
  color: #333;
}
.home-livestream-setup-title,
.home-featured-categories-title{
  float: left;
  width: 100%;
  padding: 15px 20px;
  border-bottom: 1px solid #efefef;
}
.home-livestream-setup-title h4,
.home-featured-categories-title h4{
  font-size: 16px;
  color: #333;
  float: left;
  font-weight: 500;
  font-family: 'Roboto', sans-serif;
}
.home-livestream-setup-text{
  float: left;
    width: 100%;
    text-align: center;
    padding: 20px;
}
.home-livestream-icon{
  float: left;
    width: 100%;
    text-align: center;
    margin-bottom: 20px;
}
.home-livestream-icon img{
  /* display: inline-block; */
  width: 70px;
  height: 70px;
  /* line-height: 43px; */
  /* font-size: 30px; */
  /* text-align: center; */
  padding: 14px 0;
  background: #ffecec;
  border-radius: 100%;
  /* border-bottom: 1px solid #efefef; */
}
.home-livestream-content{
  float: left;
  width: 100%;
  padding: 0;
  text-align: center;
}
.home-livestream-content p{
  font-size: 14px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    line-height: 26px;
    margin-bottom: 20px;
    color: #686f7a;
}
.home-livestream-link{
  height: 40px;
    background: #ed2a26;
    border: 0;
    border-radius: 3px;
    padding: 10px 20px;
    color: #fff;
}
.home-livestream-info{
  font-size: 12px;
  font-weight: 400;
  font-family: 'Roboto', sans-serif;
  margin-bottom: 0;
  margin-top: 15px;
  color: #686f7a;
  display: block;
}
.home-get-started{
  background: #fff;
    width: 100%;
    float: left;
    padding: 30px;
    border-radius: 3px;
    border: 1px solid #efefef;
    position: relative;
    text-align: left;
    margin-bottom: 25px;
    background-image: url('https://gambolthemes.net/html-items/cursus-new-demo/images/get-img.jpg');
}
.home-get-started h4{
  font-size: 18px;
  color: #fff;
  font-weight: 600;
  font-family: 'Roboto', sans-serif;
  margin-bottom: 13px;
}
.home-get-started p{
  font-size: 14px;
    color: #f9f9f9;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 27px;
}
.get-started-btn{
  height: 40px;
  padding: 0 20px;
  border: 0;
  border-radius: 3px;
  color: #fff;
  font-family: 'Roboto', sans-serif;
  background: #ed2a26;
  font-size: 14px;
  font-weight: 500;
}
.home-featured-categories-list span{
  float: left;
  width: 100%;
  padding: 10px 20px;
  font-size: 14px;
  font-family: 'Roboto', sans-serif;
  color: #686f7a;
  text-align: left;
}
.home-become-instuctor{
  background: #fff;
    width: 100%;
    float: left;
    padding: 30px;
    border-radius: 10px;
    border: 1px solid #efefef;
    position: relative;
    text-align: center;
    margin-bottom: 30px;
}
.home-become-instuctor h4{
  font-size: 18px;
    color: #333;
    font-weight: 600;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 13px;
}
.home-become-instuctor p{
  font-size: 14px;
    color: #686f7a;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 32px;
    line-height: 26px;
}
.home-our-student{
  margin-top: 5px;
  float: left;
    width: 100%;
}
.home-our-student h4{
  font-size: 18px;
    color: #333;
    text-align: left;
    float: left;
    margin-bottom: 20px;
}
.our-student-container{
  width: 100%;
  height: 100%;
  overflow: hidden;
  align-items: center;
  position: relative;
  align-items: center;
  display: flex;
}
.student-thought{
  display: flex;
  overflow-x: auto;
  scroll-behavior: smooth;
  gap: 20px;
  width: 100%;
}
.student-thought-card{
  background: #fff;
  width: calc(100% / 3 - 20px);
  padding: 30px;
  border-radius: 3px;
  border: 1px solid #efefef;
  transition: all .2s ease-in-out;
  flex-shrink: 0;
}
.student-thought-content{
  float: left;
  width: 100%;
  margin-bottom: 20px;
}
.student-thought-content p{
  font-size: 14px;
  font-weight: 400;
  font-family: 'roboto', sans-serif;
  line-height: 26px;
  color: #686f7a;
  text-align: left;
}
.student-thought-profile{
  float: left;
    width: 100%;
    display: flex;
    align-items: center;
}
.student-thought-profile h4{
  font-size: 16px;
  font-weight: 500;
  font-family: 'Roboto', sans-serif;
  color: #333 !important;
  margin-top: 0;
  margin-bottom: 0 !important;
  margin-left: 15px;
}
.student-thought-image{
  text-align: left;
}
.student-thought-image img{
  text-align: left;
    width: 50px !important;
    height: 50px !important;
    display: inline-block !important;
    border-radius: 100%;
    border: 2px solid #fff;
    box-shadow: 0px 2px 2px 0px rgba(0, 0, 0, 0.1);
}
