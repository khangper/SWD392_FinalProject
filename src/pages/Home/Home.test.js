import React from "react";
import { render, screen } from "@testing-library/react";
import Home from "../Home/Home";
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";

describe("Home", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });
  it("renders the Home component", async () => {
    const { container } = renderWithProviders(<Home />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});
