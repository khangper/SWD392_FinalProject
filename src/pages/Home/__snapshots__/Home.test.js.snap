// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the Home component 1`] = `
<div>
  <div
    class="home-container"
  >
    <div
      class="home-top"
    >
      <div
        class="home-left"
      >
        <section
          class="live-streams-section"
        >
          <div
            class="live-streams-header"
          >
            <h2
              class="live-streams-title"
            >
              Live Streams
            </h2>
            <a
              class="see-all-link"
              href="/all_livestream"
            >
              See all
            </a>
          </div>
          <div
            class="live-streams-container"
          >
            <button
              class="scroll-button left"
            />
            <div
              class="live-streams"
            />
            <button
              class="scroll-button right"
            />
          </div>
        </section>
        <section
          class="featured-courses-section"
        >
          <div
            class="featured-courses-header"
          >
            <h2
              class="featured-courses-title"
            >
              Featured Courses
            </h2>
            <a
              class="see-all-link"
              href="#"
            >
              See all
            </a>
          </div>
          <div
            class="featured-courses-container"
          >
            <button
              class="scroll-button left"
            />
            <div
              class="featured-courses"
            />
            <button
              class="scroll-button right"
            />
          </div>
        </section>
        <section
          class="featured-courses-section"
        >
          <div
            class="featured-courses-header"
          >
            <h2
              class="featured-courses-title"
            >
              Newest Courses
            </h2>
            <a
              class="see-all-link"
              href="#"
            >
              See all
            </a>
          </div>
          <div
            class="featured-courses-container"
          >
            <button
              class="scroll-button left"
            />
            <div
              class="featured-courses"
            />
            <button
              class="scroll-button right"
            />
          </div>
        </section>
        <section
          class="home-info-section"
        >
          <div
            class="home-info-card"
          >
            <div
              class="home-info-icon-container"
            >
              <img
                alt="icon"
                src="history.png"
              />
            </div>
            <h3>
              Go at your own pace
            </h3>
            <p>
              Enjoy lifetime access to courses on Edututs+'s website
            </p>
          </div>
          <div
            class="home-info-card"
          >
            <div
              class="home-info-icon-container"
            >
              <img
                alt="icon"
                src="expert.png"
              />
            </div>
            <h3>
              Learn from industry experts
            </h3>
            <p>
              Select from top instructors around the world
            </p>
          </div>
          <div
            class="home-info-card"
          >
            <div
              class="home-info-icon-container"
            >
              <img
                alt="icon"
                src="livestream.png"
              />
            </div>
            <h3>
              Find video courses on almost any topic
            </h3>
            <p>
              Build your library for your career and personal growth
            </p>
          </div>
          <div
            class="home-info-card"
          >
            <div
              class="home-info-icon-container"
            >
              <img
                alt="icon"
                src="screen.png"
              />
            </div>
            <h3>
              100,000 online courses
            </h3>
            <p>
              Explore a variety of fresh topics
            </p>
          </div>
        </section>
        <section
          class="popular-instructors-section"
        >
          <div
            class="popular-instructors-header"
          >
            <h2
              class="popular-instructors-title"
            >
              Popular Instructor
            </h2>
            <a
              class="see-all-link"
              href="/allinstructorlist"
            >
              See all
            </a>
          </div>
          <div
            class="popular-instructors-container"
          >
            <button
              class="scroll-button left"
            />
            <div
              class="popular-instructors"
            >
              <button
                class="scroll-button right"
              />
            </div>
          </div>
        </section>
      </div>
      <div
        class="home-right"
      >
        <section
          class="home-instructor-profile"
        >
          <div
            class="instructor-profile-image"
          >
            <img
              alt="Instructor Profile"
              src="profile-img.jpg"
            />
          </div>
          <div
            class="instructor-profile-content"
          >
            <div
              class="instructor-profile-header"
            >
              <a
                class="instructor-name"
                href="#"
              >
                John Doe
              </a>
              <div
                class="verified-badge"
              />
            </div>
            <div
              class="home-instructor-profile-title"
            >
              Web Developer, Designer, and Teacher
            </div>
            <ul
              class="social-icons-1"
            >
              <a
                class="sc-fb"
                href="#"
              >
                <img
                  src="facebook.png"
                />
              </a>
              <a
                class="sc-tw"
                href="#"
              >
                <img
                  src="x.png"
                />
              </a>
              <a
                class="sc-ln"
                href="#"
              >
                <img
                  src="linkedin.png"
                />
              </a>
              <a
                class="sc-yt"
                href="#"
              >
                <img
                  src="youtube.png"
                />
              </a>
            </ul>
            <div
              class="instructor-profile-stats"
            >
              100K Students â€¢ 15 Courses
            </div>
            <a
              class="instructor-profile-redirect"
              href="/my-instructor-profile"
            >
              Go to profile
            </a>
          </div>
        </section>
        <section
          class="home-livestream-setup"
        >
          <div
            class="home-livestream-setup-title"
          >
            <h4>
              Live Streaming
            </h4>
          </div>
          <div
            class="home-livestream-setup-text"
          >
            <div
              class="home-livestream-icon"
            >
              <div
                class="home-livestream-icon-image"
              >
                <img
                  src="livestream.png"
                />
              </div>
            </div>
            <div
              class="home-livestream-content"
            >
              <p>
                Set up your channel and stream live to your students
              </p>
              <a
                href="/add_livestream"
              >
                <button
                  class="home-livestream-link"
                >
                  Get Started
                </button>
              </a>
              <span
                class="home-livestream-info"
              >
                Info: This feature only for 'Instructor'
              </span>
            </div>
          </div>
        </section>
        <section
          class="home-get-started"
        >
          <h4>
            Get personalized recommendations
          </h4>
          <p>
            Answer a few questions for your top picks
          </p>
          <button
            class="get-started-btn"
          >
            Get Started
          </button>
        </section>
        <section
          class="home-featured-categories"
        >
          <div
            class="home-featured-categories-title"
          >
            <h4>
              Top Categories
            </h4>
          </div>
          <div
            class="home-featured-categories-list"
          >
            <span>
              <img
                src="development.png"
              />
              Development
            </span>
            <span>
              <img
                src="business.png"
              />
              Business
            </span>
            <span>
              <img
                src="it.png"
              />
              IT & Software
            </span>
            <span>
              <img
                src="design.png"
              />
              Design
            </span>
            <span>
              <img
                src="marketing.png"
              />
              Marketing
            </span>
            <span>
              <img
                src="book.png"
              />
              Personal Development
            </span>
            <span>
              <img
                src="music.png"
              />
              Music
            </span>
          </div>
        </section>
        <section
          class="home-become-instuctor"
        >
          <h4>
            Become an Instructor
          </h4>
          <p>
            Top instructors from around the world teach millions of students on Cursus. We provide the tools and skills to teach what you love.
          </p>
          <button
            class="get-started-btn"
          >
            Start teaching
          </button>
        </section>
      </div>
    </div>
    <div
      class="home-bottom"
    >
      <section
        class="home-our-student"
      >
        <h4>
          What Our Students Have Today
        </h4>
        <div
          class="our-student-container"
        >
          <button
            class="scroll-button left"
          />
          <div
            class="student-thought"
          />
          <button
            class="scroll-button right"
          />
        </div>
      </section>
    </div>
  </div>
</div>
`;

exports[`SaveCourse renders the SaveCourse component 1`] = `
<div>
  <div
    class="home-container"
  >
    <div
      class="home-top"
    >
      <div
        class="home-left"
      >
        <section
          class="live-streams-section"
        >
          <div
            class="live-streams-header"
          >
            <h2
              class="live-streams-title"
            >
              Live Streams
            </h2>
            <a
              class="see-all-link"
              href="/all_livestream"
            >
              See all
            </a>
          </div>
          <div
            class="live-streams-container"
          >
            <button
              class="scroll-button left"
            />
            <div
              class="live-streams"
            />
            <button
              class="scroll-button right"
            />
          </div>
        </section>
        <section
          class="featured-courses-section"
        >
          <div
            class="featured-courses-header"
          >
            <h2
              class="featured-courses-title"
            >
              Featured Courses
            </h2>
            <a
              class="see-all-link"
              href="#"
            >
              See all
            </a>
          </div>
          <div
            class="featured-courses-container"
          >
            <button
              class="scroll-button left"
            />
            <div
              class="featured-courses"
            />
            <button
              class="scroll-button right"
            />
          </div>
        </section>
        <section
          class="featured-courses-section"
        >
          <div
            class="featured-courses-header"
          >
            <h2
              class="featured-courses-title"
            >
              Newest Courses
            </h2>
            <a
              class="see-all-link"
              href="#"
            >
              See all
            </a>
          </div>
          <div
            class="featured-courses-container"
          >
            <button
              class="scroll-button left"
            />
            <div
              class="featured-courses"
            />
            <button
              class="scroll-button right"
            />
          </div>
        </section>
        <section
          class="home-info-section"
        >
          <div
            class="home-info-card"
          >
            <div
              class="home-info-icon-container"
            >
              <img
                alt="icon"
                src="history.png"
              />
            </div>
            <h3>
              Go at your own pace
            </h3>
            <p>
              Enjoy lifetime access to courses on Edututs+'s website
            </p>
          </div>
          <div
            class="home-info-card"
          >
            <div
              class="home-info-icon-container"
            >
              <img
                alt="icon"
                src="expert.png"
              />
            </div>
            <h3>
              Learn from industry experts
            </h3>
            <p>
              Select from top instructors around the world
            </p>
          </div>
          <div
            class="home-info-card"
          >
            <div
              class="home-info-icon-container"
            >
              <img
                alt="icon"
                src="livestream.png"
              />
            </div>
            <h3>
              Find video courses on almost any topic
            </h3>
            <p>
              Build your library for your career and personal growth
            </p>
          </div>
          <div
            class="home-info-card"
          >
            <div
              class="home-info-icon-container"
            >
              <img
                alt="icon"
                src="screen.png"
              />
            </div>
            <h3>
              100,000 online courses
            </h3>
            <p>
              Explore a variety of fresh topics
            </p>
          </div>
        </section>
        <section
          class="popular-instructors-section"
        >
          <div
            class="popular-instructors-header"
          >
            <h2
              class="popular-instructors-title"
            >
              Popular Instructor
            </h2>
            <a
              class="see-all-link"
              href="/allinstructorlist"
            >
              See all
            </a>
          </div>
          <div
            class="popular-instructors-container"
          >
            <button
              class="scroll-button left"
            />
            <div
              class="popular-instructors"
            >
              <button
                class="scroll-button right"
              />
            </div>
          </div>
        </section>
      </div>
      <div
        class="home-right"
      >
        <section
          class="home-instructor-profile"
        >
          <div
            class="instructor-profile-image"
          >
            <img
              alt="Instructor Profile"
              src="profile-img.jpg"
            />
          </div>
          <div
            class="instructor-profile-content"
          >
            <div
              class="instructor-profile-header"
            >
              <a
                class="instructor-name"
                href="#"
              >
                John Doe
              </a>
              <div
                class="verified-badge"
              />
            </div>
            <div
              class="home-instructor-profile-title"
            >
              Web Developer, Designer, and Teacher
            </div>
            <ul
              class="social-icons-1"
            >
              <a
                class="sc-fb"
                href="#"
              >
                <img
                  src="facebook.png"
                />
              </a>
              <a
                class="sc-tw"
                href="#"
              >
                <img
                  src="x.png"
                />
              </a>
              <a
                class="sc-ln"
                href="#"
              >
                <img
                  src="linkedin.png"
                />
              </a>
              <a
                class="sc-yt"
                href="#"
              >
                <img
                  src="youtube.png"
                />
              </a>
            </ul>
            <div
              class="instructor-profile-stats"
            >
              100K Students â€¢ 15 Courses
            </div>
            <a
              class="instructor-profile-redirect"
              href="/my-instructor-profile"
            >
              Go to profile
            </a>
          </div>
        </section>
        <section
          class="home-livestream-setup"
        >
          <div
            class="home-livestream-setup-title"
          >
            <h4>
              Live Streaming
            </h4>
          </div>
          <div
            class="home-livestream-setup-text"
          >
            <div
              class="home-livestream-icon"
            >
              <div
                class="home-livestream-icon-image"
              >
                <img
                  src="livestream.png"
                />
              </div>
            </div>
            <div
              class="home-livestream-content"
            >
              <p>
                Set up your channel and stream live to your students
              </p>
              <a
                href="/add_livestream"
              >
                <button
                  class="home-livestream-link"
                >
                  Get Started
                </button>
              </a>
              <span
                class="home-livestream-info"
              >
                Info: This feature only for 'Instructor'
              </span>
            </div>
          </div>
        </section>
        <section
          class="home-get-started"
        >
          <h4>
            Get personalized recommendations
          </h4>
          <p>
            Answer a few questions for your top picks
          </p>
          <button
            class="get-started-btn"
          >
            Get Started
          </button>
        </section>
        <section
          class="home-featured-categories"
        >
          <div
            class="home-featured-categories-title"
          >
            <h4>
              Top Categories
            </h4>
          </div>
          <div
            class="home-featured-categories-list"
          >
            <span>
              <img
                src="development.png"
              />
              Development
            </span>
            <span>
              <img
                src="business.png"
              />
              Business
            </span>
            <span>
              <img
                src="it.png"
              />
              IT & Software
            </span>
            <span>
              <img
                src="design.png"
              />
              Design
            </span>
            <span>
              <img
                src="marketing.png"
              />
              Marketing
            </span>
            <span>
              <img
                src="book.png"
              />
              Personal Development
            </span>
            <span>
              <img
                src="music.png"
              />
              Music
            </span>
          </div>
        </section>
        <section
          class="home-become-instuctor"
        >
          <h4>
            Become an Instructor
          </h4>
          <p>
            Top instructors from around the world teach millions of students on Cursus. We provide the tools and skills to teach what you love.
          </p>
          <button
            class="get-started-btn"
          >
            Start teaching
          </button>
        </section>
      </div>
    </div>
    <div
      class="home-bottom"
    >
      <section
        class="home-our-student"
      >
        <h4>
          What Our Students Have Today
        </h4>
        <div
          class="our-student-container"
        >
          <button
            class="scroll-button left"
          />
          <div
            class="student-thought"
          />
          <button
            class="scroll-button right"
          />
        </div>
      </section>
    </div>
  </div>
</div>
`;
